evap/evaluation/tests/test_commands.py

```diff
@@ -1,21 +1,30 @@
+from collections import defaultdict
 from datetime import datetime, date, timedelta
 from io import StringIO
+from itertools import chain
 import os
+import random
 from unittest.mock import patch
 
 from django.conf import settings
 from django.core import management, mail
+from django.db.models import Sum
 from django.test import TestCase
 from django.test.utils import override_settings
 
 from model_mommy import mommy
 
-from evap.evaluation.models import Course, Evaluation, EmailTemplate, Semester, UserProfile
+from evap.evaluation.models import (CHOICES, Contribution, Course, Evaluation, EmailTemplate, NO_ANSWER,
+    Question, Questionnaire, RatingAnswerCounter, Semester, UserProfile)
 
 
 class TestAnonymizeCommand(TestCase):
-    @patch('builtins.input')
-    def test_anonymize_does_not_crash(self, mock_input):
+    @staticmethod
+    def get_random_rating_question_type():
+        return random.choice(list(CHOICES.keys()))
```

Comment:
it seems uncritical here but generally i would avoid randomness to avoid random failures. not sure what mommy.make allows, but perhaps you can use https://docs.python.org/3/library/itertools.html#itertools.cycle somehow? this would also make sure we're using all the types.

edit: nevermind. the code does not have any conditions based on the question type, it really doesn't matter. same for the other randomness in the other tests.

Reply:
I think it's a fair point. It's probably way easier to debug if every execution runs down the same path. I made sure that we always do a `random.seed` before using random so every test run should not always be the same. I also exchanged the question type generation with that `cycle` method, seems like a much better idea there.